src_new/util.js

```diff
@@ -913,3 +913,28 @@ exports.ajaxRequest = function(url, callback, data, options) {
 		refThis.log(error);
 	}
 };
+
+// Returns mediaTypes for adUnits which are sent to prebid
+exports.getMediaTypeObject = function(nativeConfig, sizes, currentSlot){
+	var mediaTypeObject = {};
+	if(nativeConfig && nativeConfig.kgp && nativeConfig.klm){
+		var kgp = nativeConfig.kgp;
+		var klm = nativeConfig.klm;
+		var kgpv = kgp.replace(constCommonMacroForAdUnitIDRegExp, currentSlot.getAdUnitID())
```

Comment:
please use kgpv generating function from utils than duplicating code

Reply:
Done Using the common method